cscs-checks/libraries/gridtools/gridtools_base.py

```diff
@@ -0,0 +1,180 @@
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+class GridToolsCheckBase(rfm.RegressionTest):
+    def __init__(self):
+        super().__init__()
+        self.descr = 'GridTools test base'
+
+        self.valid_prog_environs = ['PrgEnv-gnu']
+        self.prebuild_cmd = ['module switch gcc/6.2.0 gcc/5.3.0']# TODO: fix this
+        self.modules = ['/scratch/snx3000/bignamic/easybuild_install/modules/all/CMake/3.12.4', 'Boost', 'cudatoolkit']
```

Comment:
We do have CMake 3.12 installed on Daint/Dom already, so you should simply do

```python
self.modules = ['CMake', 'cudatoolkit']
```

Reply:
The minimum CMake version for GT is currently 3.12.4

Reply:
I suggest installing this version through Easybuild and providing it as a standard module.

Reply:
PR created https://github.com/eth-cscs/production/pull/1137

Reply:
Done.